src/analytics/Prices.js

```diff
@@ -0,0 +1,82 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import { Col, Row } from 'reactstrap';
+
+import ErrorBoundary from 'components/ErrorBoundary';
+import { compareStrings, SortableTable } from 'components/SortableTable';
+import { cardEtchedPrice, cardFoilPrice, cardNormalPrice, cardPrice, cardStatus } from 'utils/Card';
+
+const Prices = ({ cards }) => {
+  const getValue = (collection, allowedStatus, allowedFinish) => {
```

Comment:
Does this need to be nested inside the React component? It function doesn't depend on any props, so I don't see a reason to recreate it on every render.